base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +419,52 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen;
+    int i;
+
+    if (this->m_dcGrid == NULL)
+        return;
+
+    HPEN solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
+
+    /*  BitBlt it to itself to scroll */
+    BitBlt(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top+1,
+           this->m_nPlotWidth, this->m_nPlotHeight, this->m_dcGrid,
+           this->m_rectPlot.left+this->m_nShiftPixels, this->m_rectPlot.top+1,
+           SRCCOPY);
```

Comment:
an optional coding style improvement

```suggestion
    BitBlt(this->m_dcGrid, this->m_rectPlot.left, this->m_rectPlot.top + 1,
           this->m_nPlotWidth, this->m_nPlotHeight, this->m_dcGrid,
           this->m_rectPlot.left+this->m_nShiftPixels, this->m_rectPlot.top + 1,
           SRCCOPY);
```

Reply:
You've missed one point though. I've committed the changes.